commander_frontend/script.js

```diff
@@ -1,28 +1,65 @@
+//Imports
+import { Teste } from "./modules/Teste.js";
+import { Item } from "./modules/Item.js";
+import { createItem } from "./modules/CreateElements.js";
 
+//Send Button
+var btn_send = document.getElementById('send')
 
-// fetch('http://localhost:8080')
-//   .then(response => response.text())
-//   .then(data => console.log(data))
-//   .catch(error => console.error(error));
+//Object Infos
+var namePlayer = document.getElementById('name_player');
+var date = document.getElementById('date');
+var payment = document.getElementsByName('pay-method');
+
+//Products 
+var btn_addItem = document.getElementById('add-item');
+var nameProduct = document.getElementsByName('product');
+var qtdProduct = document.getElementsByName('quantity');
+var contador = 1;
+var maxProducts = 5;
+
+//Add Product
+btn_addItem.addEventListener('click', event =>{
+  contador++;
+  event.preventDefault();
+
+  if(contador < maxProducts){
```

Comment:
Follow a pattern. If you are naming variables in English, keep everything in English.